src/win/fs.c

```diff
@@ -1313,41 +1276,47 @@ static void fs__ftruncate(uv_fs_t* req) {
 
 
 static void fs__sendfile(uv_fs_t* req) {
-  int fd_in = req->file.fd, fd_out = req->fs.info.fd_out;
+  HANDLE fd_in = req->file.hFile;
+  HANDLE fd_out = req->fs.info.hFile_out;
   size_t length = req->fs.info.bufsml[0].len;
   int64_t offset = req->fs.info.offset;
-  const size_t max_buf_size = 65536;
+  const size_t max_buf_size = 65535; /* maximum size supported by pipe write operations across a network */
```

Comment:
On re-reviewing the documentation, I'm not sure if the change is worthwhile. The documentation for `WriteFileEx` says the max might be limited 65535. The documentation for `WriteFile` says the max varies by platform but might be ~65505 bytes (x86), ~32737 (x64), ~65484 (Itanium) on some versions of Windows. With that much variance and inaccuracy, I'm just confused. I'm also not entirely sure why someone would open a pipe over a network, rather than using a socket.